logserver/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please add license header in a new file, otherwise checkstyle will be unhappy

Reply:
I think the license needs to be at the top of the file